app.py

```diff
@@ -1998,6 +1998,82 @@ def unlink_globus_identities():
+        if capabilities:
+            # Use this check in case certain user doesn't have the connection info
+            if user.connection:
+                # Note user.connection returns a list of connections (should be only one though)
+                connection_data = user.connection[0]
+                # Also get the globus_user_id
+                wp_user_meta_globus_user_id = WPUserMeta.query.filter(WPUserMeta.user_id == user.id, WPUserMeta.meta_key.like('openid-connect-generic-subject-identity')).first()
+
+                # Construct a new member dict and add to the members list
+                member = {
+                    'globus_user_id': wp_user_meta_globus_user_id.meta_value,
+                    'first_name': connection_data.first_name,
+                    'last_name': connection_data.last_name,
+                    'organization': connection_data.organization,
+                    'component': connection_data.department,
+                    'role': connection_data.title,
+                    'bio': connection_data.bio,
+                    'email': user.user_email.lower()
+                }
+                keys = get_connection_keys()
+                obj = None
+                try:
+                    globus_user = next((meta for meta in user.metas if meta.meta_key == 'globus_username'), {'meta_value': ''})
+                    if globus_user is not None:
+                        member['globus_username'] = globus_user['meta_value'] if isinstance(globus_user, (dict)) else globus_user.meta_value
+                    for k in keys:
+                        if k == 'pm':
+                            obj = next((meta for meta in connection_data.metas if meta.meta_key == connection_meta_key_prefix + 'pm'), {'meta_value': ''})
+                            if obj is not None:
+                                member['pm'] = 'Yes' if obj.meta_value == '1' else 'No'
+                        else:
+                            obj = next((meta for meta in connection_data.metas if meta.meta_key == connection_meta_key_prefix + k), {'meta_value': ''})
+                            if obj is not None:
+                                member[k] = obj['meta_value'] if isinstance(obj, (dict)) else obj.meta_value
+                except Exception as e:
+                    print(e)
+                    # print(user.user_email)
+
+                members.append(member)
+
+    return members
+
+@app.route("/downloads/members", methods=['GET'])
+@login_required
+@admin_required
+def downloads_profile():
+    context = {
+        'isAuthenticated': True,
+        'username': session['name']
+    }
+    members = get_all_members_with_all_info()
+    cvs = 'Globus Username Associated, Name, Email, Component, PM, PM email, Role, SenNet Data Via Globus, Gdrive account, GitHub, Slack, protocols.io  \n'
```

Comment:
We should have both `globus_user_id` and `globus_username` in the CSV columns.

Reply:
@yuanzhou Columns included according to the sample csv file that was provided in issue https://github.com/sennetconsortium/member-ui/issues/4 . Please confirm you still want this changed.